test/conformance/device/device_adapter_level_zero.match

```diff
@@ -21,3 +21,4 @@ urDeviceGetInfoTest.Success/UR_DEVICE_INFO_MEM_CHANNEL_SUPPORT
 urDeviceGetInfoTest.Success/UR_DEVICE_INFO_HOST_PIPE_READ_WRITE_SUPPORTED
 urDeviceGetInfoTest.Success/UR_DEVICE_INFO_MAX_REGISTERS_PER_WORK_GROUP
 urDeviceGetInfoTest.Success/UR_DEVICE_INFO_VIRTUAL_MEMORY_SUPPORT
+{{OPT}}Segmentation fault
```

Comment:
Should `Segmentation fault` be a valid outcome of those tests?

Reply:
Well, no, but that's the reality right now.

Reply:
It is okey for now